frontends/web/src/routes/account/send/feetargets.tsx

```diff
@@ -182,7 +182,9 @@ class FeeTargets extends Component<Props, State> {
                                     align="right"
                                     className={`${style.fee} ${style.feeCustom}`}
                                     disabled={disabled}
-                                    label={t('send.fee.label')}
+                                    label={t('send.feeTarget.customLabel', {
+                                        context: isBitcoinBased(coinCode) ? '' : 'eth'
```

Comment:
The opposite check makes more sense (isEthereumBased)? If we added another coin it would say 'gas price', but 'fee rate' should be the default generic text.

Reply:
Rebased and updated.